lib/tasks/bower-install.js

```diff
@@ -2,56 +2,28 @@
 
 // Runs `bower install` in cwd
 
-var Promise = require('../ext/promise');
-var Task    = require('../models/task');
+var BowerTask   = require('./bower-task');
+var _           = require('lodash');
 
-module.exports = Task.extend({
-  init: function() {
-    this.bower = this.bower || require('bower');
-    this.bowerConfig = this.bowerConfig || require('bower-config');
-  },
-  // Options: Boolean verbose
-  run: function(options) {
-    var chalk          = require('chalk');
-    var bower          = this.bower;
-    var bowerConfig    = this.bowerConfig;
-    var ui             = this.ui;
-    var packages       = options.packages || [];
-    var installOptions = options.installOptions || { save: true };
+module.exports = BowerTask.extend({
+  command: 'install',
+  startProgressMessage: 'Installing browser packages via Bower',
+  completionMessage: 'Installed browser packages via Bower.',
 
-    ui.startProgress(chalk.green('Installing browser packages via Bower'), chalk.green('.'));
+  buildOptions: function(options) {
```

Comment:
@stefanpenner @kellyselden looks difficult. Do you have any ideas how to make it more simple or reasonable?